docs/deploy.md

```diff
@@ -107,7 +107,7 @@ Prerequisites:
 Add the Sematic Helm repository to your cluster:
 
 ```shell
-$ helm repo add sematic-ai https://sematic-ai.github.io/helm-charts
+$ helm repo add sematic https://sematic-ai.github.io/helm-charts
```

Comment:
@snoshy Please confirm this is correct.

Reply:
our helm docs state

```
helm repo add sematic-ai https://sematic-ai.github.io/helm-charts
```

so we should make it consistent by changing it here or there.

Reply:
@snoshy To clarify, this did not work for me, that's why I changed it:
```
$ helm repo add sematic-ai https://sematic-ai.github.io/helm-charts
"sematic-ai" already exists with the same configuration, skipping
$
$ helm install sematic-server sematic/sematic-server -n stage -f ../charts/stage_values.yml
Error: INSTALLATION FAILED: repo sematic not found
$
$ helm repo add sematic https://sematic-ai.github.io/helm-charts
"sematic" has been added to your repositories
$
$ helm install sematic-server sematic/sematic-server -n stage -f ../charts/stage_values.yml
NAME: sematic-server
LAST DEPLOYED: Mon Mar 27 23:26:05 2023
NAMESPACE: stage
STATUS: deployed
REVISION: 1
NOTES:
Sematic dashboard url: http://stage.dev-usw2-sematic0.sematic.cloud/
```

Reply:
Clarified offline. Instead of switching `helm repo add sematic-ai` to `helm repo add sematic`, I will be switching `helm install sematic-server sematic/sematic-server` just below to `helm install sematic-server sematic-ai/sematic-server`.